### ISSUE TITLE
Feature request: One-click fill (don't show dialog box)

### ISSUE THREAD

**Issue by ckrf** [kelciour/google-translate#19]
Amazing add-in, thank you!

_Use case_: Typically I will be adding a string of vocab words all at once. I need to select the language one time but every subsequent time I just click the add-in icon and then immediately click "Start".

_Request_: I'd like a second icon or a hotkey (like clicking the icon while holding Shift) that will make it so the target field automatically gets filled without opening a dialog box to select languages, fields, extra fields, etc.

**Comment by nicgdd** [comment]
Yes please!!
One-click, or keyboard shortcut, someone, pleaaase!

That would make this add-on perfect :)

**Comment by werowe** [comment]
yes that is crucial.  we add one word at a time.  that is how we use it.  not a batch.

**Comment by werowe** [comment]
Just need to get rid of this:

` self.show()`

and hard code all of this:

        ```
self.form.sourceField.addItems(fields)
        self.form.targetField.addItems(fields)
        self.form.rmTargetField.addItems(fields)
        self.form.rmField.addItems(fields)
        self.form.mdField.addItems(fields)
        self.form.exField.addItems(fields)
        self.form.atField.addItems(fields)


```
